game_server/src/webhogg_player.rs

```diff
@@ -0,0 +1,3 @@
+pub struct WebhoggPlayer {
```

Comment:
This declaration is a duplicate of game_server/src/webhogg_game.rs::WebhoggPlayer.
However, this script is not registered into main.rs mod files xd.